packages/ckan/test/ckan.test.js

```diff
@@ -145,4 +129,25 @@ describe('@zazuko/trifid-plugin-ckan', () => {
       strictEqual(convertLegacyFrequency(`${euFreqPrefix}/WEEKLY_3`), `${euFreqPrefix}/WEEKLY_3`)
     })
   })
+
+  describe('BLW tests', () => {
+    const parser = new xml.Parser({
+      explicitArray: false,
+    })
+
+    it('should get a correct contactPoint', async () => {
+      const ckanUrl = `${getListenerURL(trifidListener)}/ckan?organization=https://register.ld.admin.ch/opendataswiss/org/bundesamt-fur-landwirtschaft-blw`
+
+      const res = await fetch(ckanUrl)
+      const body = await parser.parseStringPromise(await res.text(), { explicitArray: false })
```

Comment:
My editor complained about the usage of the call to `parseStringPromise`, saying that it only takes one argument instead of two.

I will fix it in a quick commit.